rpc/eth_api.go

```diff
@@ -698,37 +700,39 @@ func (e *PublicEthAPI) GetTransactionReceipt(hash common.Hash) (map[string]inter
 		status = hexutil.Uint(0)
 	}
 
-	res, _, err := e.cliCtx.Query(fmt.Sprintf("custom/%s/%s/%s", types.ModuleName, evm.QueryTxLogs, hash.Hex()))
-	if err != nil {
-		return nil, err
-	}
-
-	var logs types.QueryETHLogs
-	e.cliCtx.Codec.MustUnmarshalJSON(res, &logs)
-
 	txData := tx.TxResult.GetData()
+	fmt.Println(tx.TxResult.GetLog())
+
 	data, err := types.DecodeResultData(txData)
 	if err != nil {
 		return nil, err
 	}
 
+	fmt.Println(data.String())
```

Comment:
I'm seeing `internal error` coming from this when deploying a contract through rpc_test.go `TestEth_GetTransactionReceipt`

Reply:
that's probably out of gas